indico/modules/events/controllers/display.py

```diff
@@ -27,13 +27,18 @@
 class RHExportEventICAL(RHDisplayEventBase):
     @use_kwargs({
         'scope': EnumField(CalendarScope, load_default=None),
-        'detail': fields.String(load_default=None)
+        'detail': fields.String(load_default=None),
+        'export_event_series': fields.Boolean(load_default=False)
```

Comment:
Hmm what about calling the URL argument just `series` or `full_series` or `include_series`?